File: Title Case/Title Case.js
function titleCase(title, minorWords="") {
  if(title==="")return ""
  const minor_l = minorWords.split(" ").map(v => v.toLowerCase())
  return title.toLowerCase().split(" ").map((v,i) => i===0 || !minor_l.includes(v) ? v[0].toUpperCase()+v.slice(1,).toLowerCase() : v.toLowerCase()).join(" ")
}

File: Title Case/Title Case.py
def title_case(title, minor_words=''):
    minor_list = [s.lower() for s in minor_words.split(" ")]
    result = []
    for i,s in enumerate(title.split(" ")):
        s = s.lower()
        if i==0 or (s not in minor_list):
            result.append(s.title())
        else:
            result.append(s.lower())
    return " ".join(result)
